### ISSUE TITLE
Setup poiana for charts repo

### ISSUE THREAD

**Issue by leodido** [falcosecurity/test-infra#129]
**Motivation**

A new repo has been created for porting the Falco Helm charts into this organization.

Repo: https://github.com/falcosecurity/charts

**Feature**

Enable Prow on it.

**Alternatives**

There is no alternative in the world to our beloved @poiana, sorry.

**Comment by leodido** [comment]
Blocked by https://github.com/falcosecurity/charts/pull/2

**Comment by leogr** [comment]
Already done
